src/components/MediaInputs/MediaInput.spec.ct.tsx

```diff
@@ -1,6 +1,6 @@
 import { test, expect } from '@playwright/experimental-ct-react';
 
-import MediaInputs from './MediaInput';
+import MediaInputs from './MediaInput'; // Forna de importar diferente
```

Comment:
Pode evitar comentário desnecessário

```suggestion
import MediaInputs from './MediaInput';
```